services/payment_manager.js

```diff
@@ -11,4 +11,18 @@ exports.process_create_input_req = async(input_response)=>{
 }
 exports.post_create_process = async(req,resp,input_response)=>{
   resp.status(200).send(input_response)
+}
+exports.pre_process_transaction_details = async(req,resp)=>{
+  const transaction_details = await  models.Payment.findAll()
```

Comment:
whose transactions are we trying to take out here? shouldnt it be for all transactions for a user .fetch user id and find transactions of that userid